test/e2e/lib/components/find-a-domain-component.js

```diff
@@ -21,16 +21,10 @@ export default class FindADomainComponent extends AsyncBaseContainer {
 	async waitForResults() {
 		const driver = this.driver;
 		const resultsLoadingLocator = By.css( '.domain-suggestion.is-placeholder' );
-		await driver.wait(
-			function () {
-				return driverHelper
-					.isElementLocated( driver, resultsLoadingLocator )
-					.then( function ( present ) {
-						return ! present;
-					} );
-			},
-			this.explicitWaitMS * 2,
-			'The domain results loading element was still present when it should have disappeared by now.'
+		await driverHelper.waitUntilElementNotLocated(
```

Comment:
There are more changes that just adding `return await`. Is this intentional?

Reply:
Yeah, sorry for the confusion. While validating manually `return await` I noticed some obvious quick wins that I decided to commit along as I didn't want to lose track of them. They're basically all the same - applying dedicated helpers where they're missed.